stories/Range.stories.svelte

```diff
@@ -0,0 +1,16 @@
+<script>
+import Index from '@/ui/Svg/index.svelte';
+
+  import { Meta, Template, Story } from '@storybook/addon-svelte-csf'
+
+  import Range from '../src/ui/Range.svelte'
```

Comment:
1. External module imports should come first, then internal imports;
2. Instead of relative imports use the `@/` path alias;